lib/sles4sap_publiccloud.pm

```diff
@@ -475,14 +473,14 @@ sub cleanup_resource {
     my ($self, %args) = @_;
     my $timeout = bmwqemu::scale_timeout($args{timeout} // 300);
 
-    $self->run_cmd(cmd => "crm resource cleanup");
+    $self->run_cmd(cmd => 'crm resource cleanup');
 
     # Wait for resource to start
     my $start_time = time;
     while ($self->is_hana_resource_running() == 0) {
         if (time - $start_time > $timeout) {
-            record_info("Cluster status", $self->run_cmd(cmd => $crm_mon_cmd));
-            die("Resource did not start within defined timeout. ($timeout sec).");
+            record_info('Cluster status', $self->run_cmd(cmd => $crm_mon_cmd));
+            die("cleanup_resource [ERROR] Resource did not start within defined timeout. ($timeout sec).");
```

Comment:
Seems like you are enforcing a sort of die message format convention. Am I right? Does it looks good? Can you show me where/why you like this format? Are not both info already clearly available ? `[ERROR]` : die isclearly marked in red on the web interface and clearly visible in the autoinst log. The caller is also always available as you always have the callstack printed out.